lib/Cake/TestSuite/CakeTestCase.php

```diff
@@ -138,6 +97,12 @@ public function skipIf($shouldSkip, $message = '') {
 	public function setUp() {
 		parent::setUp();
 
+		if (!empty($this->fixtures)) {
+			$this->fixtureManager = $this->fixtureManager ?? CakeFixtureManager::getInstance();
+			$this->fixtureManager->fixturize($this);
```

Comment:
The fixturize() step was done inside `CakeTestRunner` before. The fixturize() method has checks already in place to make sure it doesn't do duplicate work.